templates/common/infra/bicep/core/host/container-app.bicep

```
@@ -22,7 +28,10 @@ resource app 'Microsoft.App/containerApps@2022-03-01' = {
   name: name
   location: location
   tags: tags
-  identity: { type: managedIdentity ? 'SystemAssigned' : 'None' }
+  identity: {
+    type: managedIdentityType == 'SystemAssigned' ? 'SystemAssigned' : 'UserAssigned'
```

Can this be simplified?
```suggestion
    type: managedIdentityType
```

---

haha, yes.  This was iterated on and missed the simplest solution :)